File: linguagens/pascal/pascal/imc.pas
Program Pzim ;
var p,h,imc:real;
Begin
writeln('introduza seu peso');
readln(p);  
writeln('introduza altura');
readln(h);
imc:=p/(h*h);
	if (imc<18.5)then
	writeln('abaixo do peso');
	begin
	if (imc >=18.5) and(imc<=24.9)then
	writeln('peso normal');
	end;
	begin
	if (imc =25)then
	writeln('sobrepeso');
	end;
	begin
	if (imc >=25) and(imc<=29.9)then
	writeln('pré-obeso');
	end;
	begin
	if (imc >=30) and(imc<=34.9)then
	writeln('obeso 1');
	end;
	begin
	if (imc >=35) and(imc<=39.9)then
	writeln('obeso 2');
	end;
	begin
	if (imc >=40)then
	writeln('obeso 3');
	end;
	
	
End.

File: parte prática/linguagens/C/trabalhos em C feitos em aula/imc.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>


main()
{
	setlocale(LC_ALL,"Portuguese");
	float peso,altura,imc;
	inicio:
	printf("\nintroduza seu peso\n");
	scanf("%f",&peso);
	printf("introduza sua altura\n");
	scanf("%f",&altura);
	
	imc=peso/(altura*altura);
	
	if(imc<=16){
		printf("subpeso severo! vai se alimentar dereito, seu porra");
	}else if(imc>=16.1 && imc<=19.9){
		printf("subpeso! vai se alimentar dereito, seu porra");}
	else if(imc>=20 && imc<=24.9){
		printf("peso normal! mantem assim");
		}else if(imc>=25.1 && imc<=29.9){
		printf("sobrepeso! para de se alimentar muito, seu porra");
		}else if(imc>=30 && imc<=39.9){
		printf("obeso! espero que exploda, seu porra");
		}else if(imc>=40){
		printf("obeso mórbido! vai rolar pra cóva");
		}else{printf("sla");}
		
		goto inicio;
}
